hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -147,6 +156,21 @@ def _check_date_state(self):
         for line in self:
             line._check_date_state_one()
 
+    def _prepare_domain_check_date_state_one_state_ids(self):
```

Comment:
These methods have the same problem on the naming. Please remember always to apply the same comment to the whole diff.